scripts/all-tests.sh

```diff
@@ -98,14 +98,12 @@ END
     "unit")
         case "$SHARD" in
             "lms")
-                paver test_system -s lms --extra_args="--with-flaky" || { EXIT=1; }
-                paver coverage
+                SHARD=1 paver test_system -s lms --extra_args="--with-flaky" --cov_args="-p" || { EXIT=1; }
```

Comment:
I agree that having a different definition of TEST_SUITE and SHARD would be good here.  I didn't want to have to change the groovy script in jenkins for this PR, and figured that would more naturally fall into the PR for actually changing the shards.